### DISCUSSION TITLE
Using TeslaBleHttpProxy with the Tesla Custom Integration/TeslaHttpProxy Addon in Home Assistant

### DISCUSSION THREAD

**Discussion by JasonFaulks** [wimaha/TeslaBleHttpProxy#51] [General]
Hi everyone. Has anyone tried to make the combination described in the subject line work? I have the BLE proxy working but have no idea how to line it up for connection with the Tesla Custom Integration and proxy addon. Any ideas?

**Comment by sftman18** [comment]
If your goal is getting the Tesla values/control into HA over BlueTooth, [this](https://github.com/tesla-local-control/tesla-local-control-addon) project looks like it does that.

**Reply by Raudi1** [reply]
It would be great if the projects could be merged or combined. I want to use both EVCC and HA.
